features/karaf/openhab-tp/src/main/feature/feature.xml

```diff
@@ -154,7 +154,12 @@
 
   <feature name="openhab.tp-serial-rxtx" version="${project.version}">
     <capability>openhab.tp;feature=serial;impl=rxtx</capability>
-    <bundle>mvn:com.neuronrobotics/nrjavaserial/3.14.0</bundle>
+    
+    <!-- Use "no liblockdev" version -->
+    <!-- See: https://github.com/openhab/openhab-core/pull/761 -->
+    <!-- See: https://github.com/openhab/openhab-core/issues/750 -->
+    <!-- <bundle>mvn:com.neuronrobotics/nrjavaserial/3.15.0</bundle> -->
+    <bundle>mvn:org.openhab/nrjavaserial/3.15.0.OH2</bundle>
```

Comment:
Should this be statically called, or should it be...
```
<bundle>mvn:org.openhab/nrjavaserial/${nrjavaserial.version}</bundle>
```
... like it was in the openhab-transport-serial?

Reply:
The version should be statically (I tried to already comment this in a self made review of my own changes).